data/locale/ca-ES.ini

```diff
@@ -2,13 +2,23 @@ NDIPlugin.Default="Per defecte"
 NDIPlugin.NDISourceName="Orígen NDI™"
 NDIPlugin.SourceProps.SourceName="Nom de l'origen"
 NDIPlugin.SourceProps.Bandwidth="Ample de banda"
-NDIPlugin.SourceProps.Sync="Sincronització"
+NDIPlugin.SourceProps.Sync="Audio/Video Sync"
```

Comment:
I understand that you are trying to make sure that the string property name is present in all translations, but **adding code that is wrong** seems like a worse practice than the current code's **not adding all string property names to all resource files**.

String translation implementation already falls back to English for any missing non-English string.
So, it already does exactly what you did, but without adding any code.
Same goes for any language that isn't one of the 10 that obs-ndi is translated to.

I see the argument to do this, but in my personal opinion, and as the only other reviewer, this is wrong **to me**.
To [admittedly absurdly] complete the intention of this PR's change would be to add resource files for **all** (100+?) languages with only English strings in them.
That would be a bad idea.
Similarly, I think adding English strings to non-English resource files is a bad idea.

I will approve adding German translations.
I will approve translating all strings.
I will not personally approve adding English translations to non-English languages, especially when that is, again, already the current behavior of the code without adding [wrong] code/strings.

Please either:
1. Revert adding these string property names and English strings
    -or-
2. Reasonibly translate all strings of the string property names that you are adding

Thanks!

Reply:
I reverted adding the English strings.